include/dfrn.php

```diff
@@ -1005,11 +1044,12 @@ public static function deliver($owner,$contact,$atom, $dissolve = false) {
 
 		$postvars['ssl_policy'] = $ssl_policy;
 
-		if($page)
+		if ($page) {
 			$postvars['page'] = $page;
+		}
 
 
-		if($rino>0 && $rino_remote_version>0 && (! $dissolve)) {
+		if ($rino>0 && $rino_remote_version>0 && (! $dissolve)) {
```

Comment:
Spaces before and behind the ```>``` would be nice. But like I told above, this is no blocker.